src/Unpacker.php

```diff
@@ -72,6 +73,21 @@ public function unpack(Operation $op, Result $result = null, &$links = []): Resu
             }
 
             $versionSelector = null;
+
+            foreach ($pkg->getReplaces() as $link) {
+                if ($package['dev']) {
```

Comment:
Why checking this for each iteration while it does not depend on the loop at all ? Put a `if` around the loop instead.